CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ lesser changes or bug fixes.
 
 ## [Unreleased][]
 
-* Your contribution here!
```

Comment:
Sorry, I should have specified. Can you please put the `* Your contribution here!` back in to guide future contributors? Thanks!